modules/wso2is/manifests/params.pp

```diff
@@ -0,0 +1,104 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#----------------------------------------------------------------------------
+
+class wso2is::params {
+
+  $hostname = 'localhost'
+  $mgt_hostname = 'localhost'
+
+  # Master-datasources
+  $wso2_carbon_db = {
+    name                => 'WSO2_CARBON_DB',
```

Comment:
just use the bare minimum changes required to configure the profile here, since carbon_db is not updated in this situation, you can totally get rid of that here, no need to clutter the params.pp with commented bulky data holders.